src/classes/ERR_Handler.cls

```diff
@@ -420,15 +864,29 @@ public class ERR_Handler {
         String recordURL = URL.getSalesforceBaseUrl().toExternalForm() + '/';
         if (saveResultId != null)
             recordURL += saveResultId;
-        else            
+        else
             recordURL += objId;
-                     
+
         if (recordURL.startsWith('http:'))
             recordURL = recordURL.replaceFirst('http:', 'https:');
-            
+
         return recordURL;
     }
-    
+
+    /*******************************************************************************************************
+    * @description Returns the URL of a record, if available.
+    * @param recordId The id of the record associated with the error
+    * @return The URL of the record. Warning: it may not necessarily be valid.
+    */
+    private static String getRecordURL(String recordId) {
+        String recordURL = URL.getSalesforceBaseUrl().toExternalForm() + '/' + recordId;
+
+        if (recordURL.startsWith('http:'))
```

Comment:
Curly brackets are needed, though I'm really sure the `if` is needed for this.

Reply:
This method was there originally.  I can update it  accordingly.

Reply:
Yes, sorry. If you're doing some significant work in the class and/or modifying methods, it's worth fixing legacy style errors.